qiskit_ibm_provider/ibm_backend_service.py

```diff
@@ -353,13 +350,14 @@ def job_ids(
         limit: Optional[int] = 10,
         skip: int = 0,
         backend_name: Optional[str] = None,
-        status: Optional[Union[JobStatus, str, List[Union[JobStatus, str]]]] = None,
+        status: Optional[
+            Union[Literal["pending", "completed"], List[Union[JobStatus, str]]]
+        ] = None,
         start_datetime: Optional[datetime] = None,
         end_datetime: Optional[datetime] = None,
         job_tags: Optional[List[str]] = None,
-        job_tags_operator: Optional[str] = "OR",
         descending: bool = True,
-    ) -> List[IBMJob]:
+    ) -> List[Any]:
         """Return a list of job IDs, subject to optional filtering.
```

Comment:
Double check but I'm pretty sure `job_ids` is only used for composite jobs.

Reply:
Yeah, these `job_ids()` can be removed